src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -516,16 +516,16 @@ window.qBittorrent.DynamicTable = (function() {
             return LocalPreferences.get('sorted_column_' + this.dynamicTableDivId);
         },
 
-        setSortedColumn: function(column) {
+        setSortedColumn: function(column, reverseSort = null) {
```

Comment:
Why not make it a bool with default `false`?
Also I would rename the parameter:
```suggestion
        setSortedColumn: function(column, reverse = false) {
```

Reply:
Agreed on naming 👍 

Given that `this.reverseSort` in `DynamicTable` is a `string`, I think a type of `null | string` has cleaner semantics than a `boolean`. `false` represents that we want to disable reverse sort, whereas `null` indicates that the behavior is left to the function's implementation. To maintain current behavior, we don't necessarily want to disable reverse sort by default - that's up to the function to decide.

Reply:
>Given that this.reverseSort in DynamicTable is a string, I think a type of null | string has cleaner semantics than a boolean. false represents that we want to disable reverse sort, whereas null indicates that the behavior is left to the function's implementation. To maintain current behavior, we don't necessarily want to disable reverse sort by default - that's up to the function to decide. Thus even if reverse was changed to a boolean, we'd still want to use a default value of null.

OK. Is it worth to have some code comment about it? It is not apparent that the `reverse` parameter is able to have a implementation-default state.

Reply:
I added some JSDoc describing the params expected usage. Lmk if that is clear enough.

Reply:
>I added some JSDoc describing the params expected usage. Lmk if that is clear enough.

👍